bodhi/server/models.py

```diff
@@ -3051,7 +3075,8 @@ def waive_test_results(self, username, comment=None, tests=None):
         # Ensure we can always iterate over tests
         tests = tests or []
 
-        for requirement in self._unsatisfied_requirements:
+        # these are the unsatisfied requirements
+        for requirement in self._greenwave_requirements[1]:
```

Comment:
A simple class for the requirement would avoid some mistakes and need for such explanation comments.

```python
class GreenwaveRequirements:
    def __init__(self):
        self.satisfied = []
        self.unsatisfied = []
```

Reply:
well, if we want to do that we could just use `collections.namedtuple`. I thought about it but it seemed a bit excessive for just this one case. Can change it if you like?

Reply:
Yup, sounds good, better than having to use an index. :+1: